README.md

```diff
@@ -253,7 +253,7 @@ agentops.end_session('Success')
 
 Track agents built with the Anthropic Python SDK (>=0.32.0).
 
-- [AgentOps integration example](./examples/anthropic/anthropic_example.ipynb)
+- [AgentOps integration example](examples/anthropic-sdk/anthropic_example.ipynb)
```

Comment:
Do the docs also point to the right place?

Reply:
this points to the notebook which changed routes with this rename